### PR TITLE
[REVIEW] Snmg python wrapper edit

### PR DESCRIPTION
**Pull request by Iroy30**

### CODE REVIEW

**Comment by Iroy30** [comment]
This PR includes changes needed in python for compatibility with cpp api. The only further change should be the replacement of the gdf_snmg_pagerank function in cpp/src/snmg/link_analysis/pagerank.cu by snmg cpp pagerank algorithm.

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,7 +31,7 @@
 - PR #306 Updated python test files to run pytest with all four RMM configurations.
 - PR #321 Added check routines for input graph data vertex IDs and offsets (cugraph currently supports only 32-bit integers).
 - PR #333 Various general improvements at the library level 
-
+- PR #353 Change snmg python wrapper in accordance to cpp api
```
Comment (BradReesWork) [inline CHANGELOG.md]: This now needs to be moved into 0.9 portion

File: cpp/src/snmg/link_analysis/pagerank.cu [hunk 2]
```diff
@@ -202,27 +183,27 @@ gdf_error gdf_multi_pagerank(const size_t n_gpus, gdf_column *src_ptrs, gdf_colu
         auto p = omp_get_num_threads(); 
         printf("\n Excecuting omp thread %d", i);
         /*cudaPointerAttributes attr;
-        cudaPointerGetAttributes (&attr, src_ptrs[i].data);
+        cudaPointerGetAttributes (&attr, src_col_ptrs[i]->data);
```
Comment (BradReesWork) [inline cpp/src/snmg/link_analysis/pagerank.cu]: line 179 - the print statement should be in a DEBUG block or removed before finial release

Comment (afender) [reply cpp/src/snmg/link_analysis/pagerank.cu]: I'll overwrite all this whole function in my next PR.
This one is a just doing a smoke test, I'll replace it with the actual PageRank.

**Comment by afender** [comment]
Let's get this one merged first so that I can merge/reconcile with #357 and fix the python issue in #357.
@BradReesWork we need your approval to merge.

**Pull request merged**